syncer/syncer.go

```diff
@@ -526,8 +532,14 @@ func (s *Syncer) Process(ctx context.Context, pr chan pb.ProcessResult) {
 	wg.Wait()           // wait for receive all fatal from s.runFatalChan
 
 	if err != nil {
-		syncerExitWithErrorCounter.WithLabelValues(s.cfg.Name).Inc()
-		errs = append(errs, unit.NewProcessError(err))
+		if terror.ErrSyncerCtxCanceled.Equal(err) {
```

Comment:
can we judge `utils.IsContextCanceledError(err)` here, and then don't need to transform the error like L1816-L1819

Reply:
yes, seems I should use `*Error.Delegate` so `utils.IsContextCanceledError(err)` could work

Reply:
fixed in https://github.com/pingcap/dm/pull/794/commits/38d71eeb6d28657d59375bf20339071a04087277, PTAL